presto-docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -0,0 +1,140 @@
+========
+Pushdown
+========
+
+Presto can push down the processing of queries, or parts of queries, into the
+connected data source. This means that a specific function, or other operation,
+is passed through to the underlying database or storage system for processing.
+
+The results of this pushdown can include the following benefits:
+
+* improved overall query performance
+* reduced network traffic between Presto and the data source
+* reduced load on the remote data source
+
+Support for pushdown is specific to each connector and the relevant underlying
+database or storage system.
+
+Analysis and Confirmation
+-------------------------
+
+Pushdown depends on a number of factors:
+
+* generic support for pushdown for that function in Presto
```

Comment:
This should talk explicitly about aggregation pushdown, since we do not have any other kind of function pushdown  yet. And then, there is nothing specific about particular functions from Presto perspective, is there? It's still up to connector to support particular pushdown.

Reply:
Not sure what to do with this remark in terms of actual changes to the document. Lets discuss this more in https://github.com/prestosql/presto/pull/6179